### DISCUSSION TITLE
Can anyone provide me with the Unix command to find the list of supported image formats supported by the installed version of image-magick on the Unix server.

### DISCUSSION THREAD

**Discussion by REDD003-sme** [ImageMagick/ImageMagick#4003] [Development]
Can anyone provide me with the Unix command to find the list of supported image formats supported by the installed version of image-magick on the Unix server.

**Comment by snibgo** [comment]
```
magick -list format
```

**Reply by REDD003-sme** [reply]
I have tried it a couple of ways and both seem to return the same "command not found". 
![image](https://user-images.githubusercontent.com/73709749/127672577-858c5696-250a-4c4f-a15b-d4baec61619b.png)
![image](https://user-images.githubusercontent.com/73709749/127672588-9e9a5040-c14f-4546-a876-c51493a5df42.png)

**Comment by fmw42** [comment]
Perhaps you only have ImageMagick 6. So try

`convert -list format`

If that does not work, then your ImageMagick install is faulty or you did not install it.
